src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -100,9 +104,16 @@ export class HuiDialogEditCard extends LitElement {
 
     let heading: string;
     if (this._cardConfig && this._cardConfig.type) {
-      heading = `${this.hass!.localize(
-        `ui.panel.lovelace.editor.card.${this._cardConfig.type}.name`
-      )} ${this.hass!.localize("ui.panel.lovelace.editor.edit_card.header")}`;
+      heading = computeRTL(this.hass)
+        ? `${this.hass!.localize("ui.panel.lovelace.editor.edit_card.header")} 
+           ${this.hass!.localize(
+             `ui.panel.lovelace.editor.card.${this._cardConfig.type}.name`
+           )}`
+        : `${this.hass!.localize(
+            `ui.panel.lovelace.editor.card.${this._cardConfig.type}.name`
+          )} ${this.hass!.localize(
+            "ui.panel.lovelace.editor.edit_card.header"
+          )}`;
```

Comment:
I would prefer if we would handle this correctly and break the translations, we can just add a placeholder in all existing translations to keep the same behavior.

Reply:
Done